### PR TITLE
RWAHS-375 Add authentication header

### PR DESCRIPTION
**Pull request by leftclickben**

### CODE REVIEW

**Comment by leftclickben** [comment]
This is my Apache2 configuration, which is hardcoded here for a single, specific Origin:

        Header always set Access-Control-Allow-Credentials "true"
        Header always set Access-Control-Allow-Origin "http://localhost:8888"
        Header always set Access-Control-Allow-Headers "Authorization, Origin, Accept"
        Header always set Access-Control-Allow-Methods "POST, GET, PUT, DELETE, OPTIONS"
        Header always set Access-Control-Max-Age "1"

        RewriteEngine On
        RewriteCond %{REQUEST_METHOD} OPTIONS
        RewriteRule ^(.*)$ "" [R=200,L]

**Comment by leftclickben** [comment]
Note the `Access-Control-Max-Age` above is for dev purposes, we shouldn't have that in UAT or Production.

File: public/app/config/env/development.js [hunk 1]
```diff
@@ -3,24 +3,32 @@
 
     define(
         [
+            'jquery',
```
Comment (kehh) [inline public/app/config/env/development.js]: where did jquery go - doesn't seem to be used in the diff other than `$`

**Comment by kehh** [comment]
Just the one question about an unused variable.

File: public/app/config/env/development.js (hunk 1 above)
Comment (leftclickben) [inline public/app/config/env/development.js]: Yep good catch

**Pull request merged**